web/src/cypress/integration/profile.ts

```diff
@@ -54,13 +60,15 @@ function testProfile(): void {
     return cy
       .createService({ name })
       .then((svc: Service) => {
-        return cy
-          .createEPStep({
-            epID: svc.epID,
-            targets: [{ type: 'user', id: profile.id }],
-          })
-          .engineTrigger()
-          .then(() => svc.id)
+        return (
+          cy
+            .createEPStep({
+              epID: svc.epID,
+              targets: [{ type: 'user', id: profile.id }],
+            })
+            // .engineTrigger()
```

Comment:
Should we remove this comment?

Reply:
Oh, this was causing tests to fail for me locally, but I don't think it was for Caza. I'll uncomment out and see what happens on CI. It also looks like there's a playwright test I need to update as well

Reply:
This is needed, so the test doesn't hang waiting for the engine to run in the background